files/icons/awesome6.toml

```diff
@@ -47,6 +47,7 @@ net_bridge = "\uf0e8"
 net_down = "\uf019"
 net_loopback = "LO"
 net_modem = "\uf095"
+net_strength = "\uf012"
```

Comment:
It's unfortunate that font awesome free has no "no signal" icon, but we can at least use the cross symbol here. Something like

```toml
net_strength = [
        "<x symbol>",
        "\uf012",
        "\uf012",
        "\uf012",
        "\uf012",
        "\uf012",
]
```

Reply:
The `network_type` has "×" (and critical state) if the signal is zero or less, so if you have icon + network type then it'll be conveyed. I think that way we see × after signal icon instead of just × which won't have context. I think that's better, what do you think?

Reply:
Preview of current state when the network is zero (tested by disabling the SIM)
![image](https://user-images.githubusercontent.com/46195892/236498532-722c9013-ca8e-4b0d-8dcd-0d0650a24449.png)

Reply:
I think that looks good. @MaxVerevkin, I'll be traveling for the next 2 weeks if you'd like to take over the reviews/merge.